vectordb_bench/backend/clients/redis/redis.py

```diff
@@ -103,20 +102,29 @@ def insert_embeddings(
         """Insert embeddings into the database.
         Should call self.init() first.
         """
+
+        batch_size = 100 # Adjust this as needed, but don't make too big
```

Comment:
Let's start with a slightly more aggressive value which should be beneficial for throughput.. say 1000!